### DISCUSSION TITLE
Problems with API since 29/6/24 AM AEST

### DISCUSSION THREAD

**Discussion by craigcurtin-dev** [amberelectric/public-api#208] [General]
@madpilot - SInce this morning my custom calls (Node Red) to the API have been failing with a 502 - the call looks like this

"https://api.amber.com.au/v1/sites/xxxxxx/prices?startDate=2024-06-29&endDate=2024-06-30&resolution=30"

This has been failing since this morning - no changes on my end

Wherease my 5 min calls are still working fine

https://api.amber.com.au/v1/sites/XXXprices/current?resolution=5

Site ID is the same in both calls 

Has something changed in preparation for the rate/tarriff changes on 1/7 ?

FYI - i am on the Ausgrid Two way trial in Syndey.

Craig

**Comment by madpilot** [comment]
I hope not because that would be bad news bears.

I will go have a look.

**Comment by madpilot** [comment]
Good super sleuthing! I'll fix it up.

**Comment by madpilot** [comment]
Fixed. Thanks for reporting

**Comment by craigcurtin-dev** [comment]
Thanks for the fast repair time Myles - very much appreciated.

Enjoy the rest of your weekend

Craig

**Comment by jeffborg** [comment]
I've been seeing 502 errors around midnight in evcc fetching the forecasts at this endpoint https://api.amber.com.au/v1/sites/SITE_ID/prices?resolution=30&endDate=2024-07-04

The endDate is set to today + 2 days

**Comment by craigcurtin-dev** [comment]
That was resolved by @Madpilot the same night (Saturday) that I raised the issue

You might need to troubleshoot a bit further – also refer to the post I did in the discussion thread for my query that was not working and now is

Craig

**Reply by jeffborg** [reply]
I don't think it's fully resolved as yet, took almost 1/2 hour this morning for the 502's to stop

evcc-5b5f675b6-p5s9b evcc [amber ] ERROR 2024/07/02 00:08:58 unexpected status: 502 (Bad Gateway)
evcc-5b5f675b6-p5s9b evcc [amber ] ERROR 2024/07/02 00:09:55 unexpected status: 502 (Bad Gateway)
evcc-5b5f675b6-p5s9b evcc [amber ] ERROR 2024/07/02 00:10:50 unexpected status: 502 (Bad Gateway)
evcc-5b5f675b6-p5s9b evcc [amber ] ERROR 2024/07/02 00:11:51 unexpected status: 502 (Bad Gateway)
evcc-5b5f675b6-p5s9b evcc [amber ] ERROR 2024/07/02 00:12:55 unexpected status: 502 (Bad Gateway)
evcc-5b5f675b6-p5s9b evcc [amber ] ERROR 2024/07/02 00:13:43 unexpected status: 502 (Bad Gateway)
evcc-5b5f675b6-p5s9b evcc [amber ] ERROR 2024/07/02 00:14:52 unexpected status: 502 (Bad Gateway)
evcc-5b5f675b6-p5s9b evcc [amber ] ERROR 2024/07/02 00:15:53 unexpected status: 502 (Bad Gateway)
evcc-5b5f675b6-p5s9b evcc [amber ] ERROR 2024/07/02 00:16:50 unexpected status: 502 (Bad Gateway)
evcc-5b5f675b6-p5s9b evcc [amber ] ERROR 2024/07/02 00:17:53 unexpected status: 502 (Bad Gateway)
evcc-5b5f675b6-p5s9b evcc [amber ] ERROR 2024/07/02 00:19:01 unexpected status: 502 (Bad Gateway)
evcc-5b5f675b6-p5s9b evcc [amber ] ERROR 2024/07/02 00:19:59 unexpected status: 502 (Bad Gateway)
evcc-5b5f675b6-p5s9b evcc [amber ] ERROR 2024/07/02 00:21:00 unexpected status: 502 (Bad Gateway)
evcc-5b5f675b6-p5s9b evcc [amber ] ERROR 2024/07/02 00:21:49 unexpected status: 502 (Bad Gateway)
evcc-5b5f675b6-p5s9b evcc [amber ] ERROR 2024/07/02 00:22:59 unexpected status: 502 (Bad Gateway)
evcc-5b5f675b6-p5s9b evcc [amber ] ERROR 2024/07/02 00:23:49 unexpected status: 502 (Bad Gateway)
evcc-5b5f675b6-p5s9b evcc [amber ] ERROR 2024/07/02 00:25:02 unexpected status: 502 (Bad Gateway)
evcc-5b5f675b6-p5s9b evcc [amber ] ERROR 2024/07/02 00:25:49 unexpected status: 502 (Bad Gateway)
evcc-5b5f675b6-p5s9b evcc [amber ] ERROR 2024/07/02 00:26:45 unexpected status: 502 (Bad Gateway)
evcc-5b5f675b6-p5s9b evcc [amber ] ERROR 2024/07/02 00:28:02 unexpected status: 502 (Bad Gateway)
evcc-5b5f675b6-p5s9b evcc [amber ] ERROR 2024/07/02 00:29:02 unexpected status: 502 (Bad Gateway)
evcc-5b5f675b6-p5s9b evcc [amber ] ERROR 2024/07/02 00:29:40 unexpected status: 502 (Bad Gateway)

**Reply by madpilot** [reply]
That's an unrelated issue. Are those timestamps UTC or local time?

**Reply by jeffborg** [reply]
NSW Sydney time @madpilot
